core/domain/feedback_services.py

```diff
@@ -266,7 +282,7 @@ def enqueue_feedback_message_email_task(user_id):
 
 
 def get_feedback_message_references(user_id):
-    model = feedback_models.UnsentFeedbackEmailModel.get(user_id, strict=True)
+    model = feedback_models.UnsentFeedbackEmailModel.get_by_id(user_id)
```

Comment:
Ohh, I was initially aiming for some other implementation but I finalised on other. I forgot to revert back this change.

Thanks